package.json

```diff
@@ -102,6 +101,6 @@
     "prettier": "^3.5.2",
     "semantic-release": "^24.2.3",
     "tempy": "^3.1.0",
-    "typescript": "^5.8.2"
```

Comment:
I realised that `typescript` and `tsconfig.json` are only used by `dtslint`, because of this line: https://github.com/keithamus/sort-package-json/blob/1c142c61f4cd370fc3b17f150927ed4d82efa6f3/package.json#L49

TSTyche does not require TypeScript to be installed. And also it sets strict and modern default compiler options: https://tstyche.org/project/compiler-options#default-compiler-options